tests/test_digest_inputs.py

```
@@ -0,0 +1,550 @@
+        _seed_evergreen_revision(
+            conn, object_id=f"evg-s-{i}", version=1, change_type="created",
+            derived_at=as_of - timedelta(hours=2),
+        )
+    conn.close()
+    result = collect_digest_inputs(
+        vault_dir, "research-tech", as_of=as_of, config=utc_config
+    )
+    assert result.pipeline_state.unsynthesized_evergreens == 6
+    threshold_clusters = result.pipeline_state.clusters_at_threshold
+    assert len(threshold_clusters) == 1
+    cid, label, count, stale = threshold_clusters[0]
```

Ruff (0.15.12)

[warning] Unpacked variable `label` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)